### DISCUSSION TITLE
Call for Features tool feedback

### DISCUSSION THREAD

**Discussion by OBWANDO** [o3de/community#113] [General]
There is a new tool for us to use so that we can get our features aligned and focused.
The original thread can be found here: https://github.com/o3de/o3de/issues/6821

I've built an initial tool that can help speed up the process here: 
https://o3de.github.io/community/features/form.html

Source here: https://github.com/o3de/community/tree/main/features

I'm looking for feedback and ideas on how we can make it easier, more efficient, and usable.
The end result is for us to have a table that shows the state of our features and where we need to put some community effort. It also allows developers to understand the maturity of the subsystems of O3DE and where they can build upon.

![image](https://user-images.githubusercontent.com/294136/153840803-8877f23d-89f3-4c6b-8d75-07a8253161e1.png)

**Comment by lmbr-pip** [comment]
Can we add documentation (link or otherwise) to cover:
* How to update? Believe currently its a PR to json file in sig-community
* Rubric/Legend about common expectation for labels? Want to ensure theres a clear set of expectations about what 'volative' means in relation to the code column etc. What is the expectation for a Platform to be set? Build on the platform, run on the platform? Run on the platform in release with performant code?

**Comment by OBWANDO** [comment]
Update is to use the tool, then use the "Download JSON" button and do a PR to the folder below in the repo with the same exact name as the SIG Json file.
https://github.com/o3de/community/tree/main/features/sigjson  

example, if you are working on sig-network-features.json you will rename your downloaded file to match and PR it.
It may take up to 10 minutes for it to go live. And if you arent ready to PR, you can always save the Json file locally and load it back up to continue your work and then save it when you are ready to PR.

I will be adding more automation to do this for everyone as I get the GitHub auth bound into it. Then you should be able to generate a PR with a single click.

Documentation and tooltips added explaining each column and entry for each cell.
https://github.com/o3de/community/blob/main/features/README.md

Let me know if there are more ideas or refinements to the data itself.
